Python/omevv/v1/scripts/CreateRepoProfile.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import sys
+import time
+import warnings
+from webbrowser import get
+
+import requests,json,base64
+warnings.filterwarnings("ignore")
+
+parser = argparse.ArgumentParser(
+    description="Python script using OME api to create Repository profile"
+);
+parser.add_argument('-ip', help='OMEVV IP address', required=True);
+parser.add_argument('-ome_user', help='OME username', required=True);
+parser.add_argument('-ome_pswd', help='OME password', required=True);
+parser.add_argument('-omevv_user', help='OMEVV username', required=True);
+parser.add_argument('-omevv_pswd', help='OMEVV password', required=True);
+parser.add_argument('-profilename', help='repository profile name', required=True);
+parser.add_argument('-description', help='repository profile description', required=False);
+parser.add_argument('-profileType', help='repository profile type', required=True);
```

Comment:
I think options can be provided for this parameter as well. I think its "Firmware" or "Driver" but please do check.